src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -640,7 +642,8 @@ TEST_F(TransportManagerImplTest, UpdateDeviceList_AddNewDevice) {
 
 struct DeviceInfoComparator {
   const transport_manager::DeviceInfo& val_;
-  DeviceInfoComparator(const transport_manager::DeviceInfo& val) : val_(val) {}
+  explicit DeviceInfoComparator(const transport_manager::DeviceInfo& val)
+      : val_(val) {}
 
   bool operator()(const transport_manager::DeviceInfo& val) const {
     return val_.name() == val.name() &&
```

Comment:
@vlantonov What do you think about operator `==` definition for `DeviceInfo`? May be it's worth to define it?